pkg/operator/ceph/pool/controller.go

```diff
@@ -422,23 +422,44 @@ func configureRBDStats(clusterContext *clusterd.Context, clusterInfo *cephclient
 	logger.Debug("configuring RBD per-image IO statistics collection")
 	namespaceListOpt := client.InNamespace(clusterInfo.Namespace)
 	cephBlockPoolList := &cephv1.CephBlockPoolList{}
-	var enableStatsForPools []string
+	var enableStatsForCephBlockPools []string
```

Comment:
Unit tests still needed, you could likely just add more testing to `TestConfigureRBDStats()`.